jbmc/src/java_bytecode/java_bytecode_language.h

```diff
@@ -59,6 +60,8 @@ Author: Daniel Kroening, kroening@kroening.com
   "                              never initialize reference-typed parameter to the\n" /* NOLINT(*) */ \
   "                              entry point with null\n" /* NOLINT(*) */ \
   " --java-max-vla-length N      limit the length of user-code-created arrays\n" /* NOLINT(*) */ \
+  " --load-containing-class-only only load the class containing the " \
+  "                              method under test and the library classes" \
```

Comment:
How does this differ from lazy loading? Obviously I'm not a Java expert, but as a mere user I'm pretty confused by this vast array of options.

Reply:
Lazy-loading will load all the classes that are needed to analyse the program. This option is more aggressive as it will cut out anything that leaves the class that contains the method under test. It's a wild over-approximation but it's an easy way of analysing blocks that would otherwise be intractable by jbmc.

Reply:
Hmm, there is all the work that @hannes-steffenhagen-diffblue has done (7952f2c21637 and related commits). Is doing it on a by-class level a much better approach for Java? Also, without your explanation above I would not have guess the semantics of this new option.

Reply:
@tautschnig The documentation is a bit sparse, I can easily agree with that. It's mostly the fault of the `--help` string format which we don't want to make to heavy. Where would a good place be to write an extended description?
At this point it's not clear how well a per-class approach would work. It's an experimental feature (I will document is as such).

Reply:
> Is doing it on a by-class level a much better approach for Java?

Reply:
Near-duplicate of test-gen's single-function-only option? Perhaps both should be generalised into a function whitelist?